pandas/tests/io/json/test_pandas.py

```diff
@@ -1262,3 +1262,12 @@ def test_index_false_error_to_json(self, orient):
                "'orient' is 'split' or 'table'")
         with pytest.raises(ValueError, match=msg):
             df.to_json(orient=orient, index=False)
+
+    @pytest.mark.parametrize('orient', ['split', 'table'])
+    def test_index_false_from_json_to_json(self, orient):
```

Comment:
Do we have adequate roundtrip coverage on `index=True` as well? Might not hurt to expand the parametrization here if not already covered

Reply:
I have added parametrization test also for `index=True`.